internal/indexmanagement/scripts.go

```diff
@@ -50,6 +51,31 @@ def deleteIndices(index):
     sys.stdout = original_stdout
     return False
 
+#here namespace is either a string or array of strings, minAge is also a string in the unit of days, e.g., '3d'
+def deleteByQuery(index, namespace, minAge):
+  defaultAge='7d'
```

Comment:
This should be an env var, which we pass to the k8s cronjob. Just in case a user wants to overwrite this temporarily for all delete-by-query calls.